src/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mochaExplorer.files": "**/*.test.js",
+  "mochaExplorer.require": "esm",
+  "[vue]": {
+    "editor.defaultFormatter": "octref.vetur"
```

Comment:
What’s different/better about this formatter?

Reply:
It understands `vue` files, and it allows you to pick linter/formatters for the different sections. It also comes with support for snippets and such. I believe it's the official vuejs tool for vscode.